webapp/channels/src/components/admin_console/cluster_table_container.tsx

```diff
@@ -1,15 +1,22 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React from 'react';
+import React, {MouseEvent} from 'react';
 
+import {ClusterInfo} from '@mattermost/types/admin';
 import {getClusterStatus} from 'actions/admin_actions.jsx';
 import LoadingScreen from '../loading_screen';
 
 import ClusterTable from './cluster_table';
 
-export default class ClusterTableContainer extends React.PureComponent {
-    constructor(props) {
+interface State {
+    clusterInfos: ClusterInfo[] | null;
+}
+
+export default class ClusterTableContainer extends React.PureComponent<object, State> {
+    interval: NodeJS.Timer | null;
+
+    constructor(props: object) {
```

Comment:
```suggestion
    constructor(props) {
```

You can leave this empty ?

Reply:
Leaving it empty gives following error

```Parameter 'props' implicitly has an 'any' type```

Reply:
pass it as Props which is just null as